### ISSUE TITLE
Launch crash for unavailable iOS framework dependencies

### ISSUE THREAD

**Issue by BlackWolf** [leavez/cocoapods-binary#39]
I recently tried to use `cocoapods-binary` on the `Auth0` pod. This caused the following launch crash for me on iOS 11 and earlier:

```
dyld: Library not loaded: /System/Library/Frameworks/AuthenticationServices.framework/AuthenticationServices
  Referenced from: /Users/username/Library/Developer/CoreSimulator/Devices/312557D4-8701-48F5-85CB-B1A0E4359B15/data/Containers/Bundle/Application/6E9A7D9E-A337-42F2-B0ED-1B2CDFB8DA58/AppName.app/AppName
  Reason: image not found
```

It seems this is because the iOS framework `AuthenticationServices`, which is available only on iOS12, is an optional dependency of Auth0. Looking at the Auth0 source, they use the following piece of code:

```
#if canImport(AuthenticationServices)
import AuthenticationServices
#endif
```

Looking at the project, it seems that `-framework AuthenticationServices` is added to the `OTHER_LDFLAGS`. When I removed this, it worked, but that's of course just a workaround, since it will be readded on every `pod install`.

**Comment by leavez** [comment]
Cocoapods-binary will preserve the framework setting of the original podspec. It seems a bug of Auth0. 

https://github.com/auth0/Auth0.swift/pull/253

@BlackWolf Could you try the least version of auth0 and confirm it's not a problem of this plugin?

**Comment by BlackWolf** [comment]
yes, problem is fixed starting with Auth0 v1.14.1. Thanks!
